### PR TITLE
Send expected errors to expected bucket

### PR DESCRIPTION
**Pull request by jridgewell**
I want them out of my error logs.

### CODE REVIEW

**Comment by jridgewell** [comment]
> Please update dashboards.

@erwinmombay, how do I do that? Actually, you'll probably need to do that since I don't have admin permissions.

**Comment by erwinmombay** [comment]
@jridgewell will do.

**Comment by jridgewell** [comment]
Merge afterwards, please.

File: tools/errortracker/errortracker.go [hunk 1]
```diff
@@ -149,6 +149,9 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		errorType += "-canary"
 		isCanary = true;
 	}
+	if r.URL.Query().Get("ex") == "1" {
```
Comment (cramforce) [inline tools/errortracker/errortracker.go]: Please increment version number in app.yaml

Comment (jridgewell) [inline tools/errortracker/errortracker.go]: Done.

**Comment by erwinmombay** [comment]
ive updated the filters on the metrics

**Pull request merged**